.github/workflows/safer-golangci-lint.yml

```
@@ -43,11 +51,12 @@ on:
     branches: [main, master]
 
 env:
-  GOLINTERS_VERSION: 1.46.2
+  GO_VERSION: 1.19
```

Go 1.19 changed `gofmt` output and we have PRs that will be merged after fuzzing, so lets use Go 1.18 for now and switch after PRs are merged.

```suggestion
  GO_VERSION: 1.18
```